src/utils.test.js

```diff
@@ -0,0 +1,51 @@
+import {
+    getPayPalWindowNamespace,
+    getBraintreeWindowNamespace,
+    hashStr,
+} from "./utils";
+
+describe("testing getPayPalWindowNamespace util function", () => {
+    const paypal = {
+        Buttons: jest.fn(),
+    };
+
+    beforeAll(() => {
+        window.paypal = paypal;
+    });
+
+    test("should return the paypal namespace", () => {
+        expect(getPayPalWindowNamespace("paypal")).toEqual(paypal);
```

Comment:
I think you could use `window.paypal` directly and eliminate the need for the extra variable.

```js
    beforeAll(() => {
        window.paypal = {
            Buttons: jest.fn(),
        };
    });
```

```suggestion
        expect(getPayPalWindowNamespace("paypal")).toEqual(window.paypal);
```